suite-common/wallet-types/src/fiatRates.ts

```diff
@@ -27,3 +31,26 @@ export interface CoinFiatRates extends TickerId {
     current?: CurrentFiatRates;
     lastWeek?: LastWeekRates;
 }
+
+export type FiatRateKey = string & { __type: 'FiatRateKey' };
```

Comment:
I :heart: to see Nominals. 

I suggest to add 


```ts
const asFiatRateKey = (value: string): FiatRateKey => value as FiatRateKey;
```

So we don't have to do typecasting all over the place and its hidden in one place.

Reply:
As other functions are using it like this it feels more like some general improvement with casting, can you mark it for some code improvement session

Reply:
I like to discuss this more, but for now it is not blocking the merge. Feel free to resolve if you really don't want to fix it now.